searx/engines/duckduckgo_extra.py

```diff
@@ -69,28 +73,68 @@ def request(query, params):
         args['p'] = safe_search  # "-1", "1"
 
     logger.debug("cookies: %s", params['cookies'])
+
+    path = 'i'
+    if ddg_category == 'videos':
```

Comment:
This should be changed to a map from category to url path to make the code easier to read.